ra/ra.go

```diff
@@ -1916,15 +1916,19 @@ func (ra *RegistrationAuthorityImpl) AdministrativelyRevokeCertificate(ctx conte
 }
 
 // DeactivateRegistration deactivates a valid registration
-func (ra *RegistrationAuthorityImpl) DeactivateRegistration(ctx context.Context, reg core.Registration) error {
-	if reg.Status != core.StatusValid {
-		return berrors.MalformedError("only valid registrations can be deactivated")
+func (ra *RegistrationAuthorityImpl) DeactivateRegistration(ctx context.Context, reg *corepb.Registration) (*emptypb.Empty, error) {
+	// Error if the request is nil, there is no account key, IP address, or regID
+	if reg == nil || len(reg.Key) == 0 || len(reg.InitialIP) == 0 || reg.Id == 0 {
```

Comment:
I know this is just carried forward from the existing `registrationValid`, but I'm pretty sure the only one of these we actually care about is `reg.ID`, as it is used on line 1927.

Reply:
Thanks, done!